### PR TITLE
Restyled select and input to the spec image

### LINKED ISSUE #5972: Separate input and select styles using Aphrodite
The goal for this issue would be to separate input and select styles (which are currently coupled together).  [Per our style guidelines](https://github.com/brave/browser-laptop/blob/master/docs/style.md), solving this issue would involve rolling the input and select elements into an Aphrodite component.

- [x] Text entry field (editable)
- [x] Text entry field (focus outline)
- [x] Text entry field (locked/fixed value)
- [x] drop-down menu style
- [x] preferred drop down arrow style
- [x] flatten out the orange button
- [x] update about:styles

Here are some spec images:

## Text entry field (editable)
image

image

## Text entry field (focus outline)
image
 (use a 2 px radius, and "@statsBlue: #0796fa;")

## Text entry field (locked/fixed value)
image

image
(use variable color for fill: @veryLightGray: rgb(250, 250, 250); )

## drop-down menu style

image

image

## preferred drop down arrow style
image

We can switch to a different FA triangle
image

## and this will flatten out the orange button
image

image

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Closes #5972
The SVG caret down will be replaced with the sprite later (#5891)

Test Plan: in https://github.com/brave/browser-laptop/pull/6084/commits/fd0c10af0de5e76fb4bbe803a907f168ba4414d2

Auditors:

screenshot 2016-12-08 14 53 33

Into
screenshot 2016-12-11 19 01 19
screenshot 2016-12-08 15 36 51

It seems there is no property such as `outline-radius`.

screenshot 2016-12-08 15 44 19

### CODE REVIEW

**Comment by luixxiul** [comment]
If there is (and should be) a better way to handle radius inside the input form, please edit the commit, thanks.

**Comment by bradleyrichter** [comment]
sweet!

I do see that we need one more style difference between editable text entry fields and non-editable. (can be the same but light grey BG instead of white.)

**Comment by luixxiul** [comment]
@bradleyrichter even `@veryLightGray` in variable.less is too dark on about:preferences, though I'm not sure if `#fefefe` makes difference.

**Comment by luixxiul** [comment]
be2a185 fixes

screenshot 2016-12-10 16 51 33

into
screenshot 2016-12-10 16 51 40

**Comment by luixxiul** [comment]
With 21a13ef

screenshot 2016-12-10 17 24 44

File: less/forms.less [hunk 1]
```diff
@@ -22,7 +22,8 @@ select {
   width: 100%;
 }
 
-select.form-control {
+select.form-control,
+.bookmarkForm select {
```
Comment (luixxiul) [inline less/forms.less]: I know this is not an elegant solution; general properties for `select` and `input` should be specified at first, followed by more specific properties for `.form-control`, etc.

**Comment by luixxiul** [comment]
With f21ff68

screenshot 2016-12-11 18 55 55

**Comment by bradleyrichter** [comment]
@luixxiul These look good. The one thing I see that is still wrong is that the form elements in the bookmark hanger should be taller to match the prefs pages.

**Comment by luixxiul** [comment]
@bradleyrichter Should it be like this?

screenshot 2016-12-12 4 42 55

**Comment by bradleyrichter** [comment]
yes. perfect.

**Comment by luixxiul** [comment]
OK, updated.

**Comment by bsclifton** [comment]
Will check this out soon (I promise!)

**Comment by bradleyrichter** [comment]
@bsclifton Candidate for 13.0? Or wait?

**Comment by bsclifton** [comment]
@bbondy since these are only style changes, if we test appropriately before accepting, could we pull this into 0.13.0?

**Comment by luixxiul** [comment]
Does anyone know any hack to fix this?

screenshot 2016-12-08 15 36 51

because outline-radius does not exist.

**Comment by bsclifton** [comment]
Let's keep this for 0.13.1 :smile:

**Comment by luixxiul** [comment]
CI passed: https://travis-ci.org/brave/browser-laptop/builds/190889920

**Comment by cezaraugusto** [comment]
look good but needs rebase

**Comment by bsclifton** [comment]
I've been very hesitant to review this PR because of all of the time cost associated with bugs and regressions that style updates introduce.  Per our reviewer guidelines, I'm going to close this issue:
https://github.com/brave/browser-laptop/wiki/Reviewer-guideline

> 6. Front end: If styles are being added, make sure it is using Aphrodite.

I'll update the original issue to capture this requirement

**Comment by luixxiul** [comment]
OK but please make sure that the guideline was published 8 days ago while the PR was initially opened 1 month ago

**Comment by bsclifton** [comment]
@luixxiul I understand.  I definitely appreciate the work, but the cost of reviewing adhoc style changes is orders of magnitude higher than reviewing code which has tests.

**Pull request closed without merging**